hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -4306,18 +4306,18 @@ public static boolean isAclEnabled(Configuration conf) {
       ROUTER_PREFIX + "interceptor.allow-partial-result.enable";
   public static final boolean DEFAULT_ROUTER_INTERCEPTOR_ALLOW_PARTIAL_RESULT_ENABLED = false;
 
+  /** Subcluster timeout allowed by Router.  **/
```

Comment:
Avoid moving this

Reply:
Thank you very much for your help to review the code, I will modify the code.